source/_integrations/zha.markdown

```diff
@@ -575,58 +575,61 @@ After restoring a Home Assistant backup, you can reconfigure ZHA or migrate to a
-      - Use a USB 2.0 port or a powered USB 2.0 hub,
-      - Keep the Zigbee stick away from USB 3.0 devices.
-7. Start the backup restore process:
-    - Select the new Zigbee radio from the list of serial ports and select **Next**.
-    - A migration can be resumed if a reboot is required, such as when troubleshooting or if new hardware is plugged in.
-      - To resume, go to **Network Settings**, select **Re-configure the current radio**, choose the new radio and proceed.
-8. Under **Network Formation**, select **Restore an automatic backup**.
-9. Under **Restore Automation Backup**, choose the latest automatic backup and select **Submit**.
-10. If the new radio requires overwriting the IEEE address (the unique MAC address), you will see the prompt for **Overwrite Radio IEEE Address**.
-    - Check the **Permanently replace the radio IEEE address** box and click **Submit**.
-      - Selecting this option is required for the migration process to complete successfully.
-      - Overwriting the IEEE address may take a while.
-    - Both the old and new Zigbee Coordinators will now have the same Zigbee IEEE address.
-    - You should not operate the old adapter in the same area unless you change its Zigbee IEEE address.
-    - If you do not migrate the Zigbee IEEE address from the old Zigbee Coordinator radio, you will have to reconnect many of your devices to keep them working.
-11. Finally, a **Success!** message should pop up with information that all options were successfully saved.
-    - Select **Finish** to confirm.
+   - An automatic backup will be performed, the Zigbee adapter will be reset, and the backup will be automatically restored.
+   - For combined Z-Wave and Zigbee adapters, like the HUSBZB-1 adapter, only the Zigbee radio portion is reset.
+   - You may now unplug the old adapter, or you may leave the old adapter plugged in (for example, if the adapter is a combined Z-Wave adapter).
+4. Under **Migrate or re-configure**, select **Migrate to a new adapter**.
+5. Plug in the new Zigbee adapter.
+   - To minimize interference:
+     - Use a USB extension cable.
+     - Use a USB 2.0 port or a powered USB 2.0 hub.
+     - Keep the Zigbee stick away from USB 3.0 devices.
+6. Select the new Zigbee adapter from the list of serial ports and select **Submit**.
+   - **Troubleshooting**: A migration can be resumed if a reboot is required, such as when troubleshooting or if new hardware is plugged in.
+   - To resume, go to **Network Settings**, select **Re-configure the current adapter**, choose the new adapter and proceed.
+7. Choose what backup to use for migration:
+   - **Option 1**: To use the backup that was created during this migration, select **Migrate automatically (recommended)**.
+     - This is the quickest way to complete the migration.
+   - **Option 2**: To restore a specific, older backup, select **Advanced migration** instead.
+     - This will let you select a backup of your choice.
```

Comment:
optional: We could also add another bullet point for the "Advanced migration" about being able to keep existing network settings. You rarely wanna do that when "migrating" adapters, mostly only when re-configuring the current adapter, but it's still an option.

Reply:
I think migrating to a *new* adapter by keeping the network settings of the new adapter is an unusual use case. We document this as a migration path from Z2M to ZHA (since ZHA will slowly find devices over time) but that would apply to setting up a new ZHA instance, not to migrating an existing one.